java-tools/OsmAndServer/src/main/java/net/osmand/server/api/services/WikiService.java

```diff
@@ -81,16 +85,36 @@ public FeatureCollection getImagesById(long id, double lat, double lon) {
 		return new FeatureCollection(features.toArray(new Feature[0]));
 	}
 	
-	public FeatureCollection getWikidataData(String northWest, String southEast, String lang, Set<String> filters) {
+	public FeatureCollection getWikidataData(String northWest, String southEast, String lang, Set<String> filters, int zoom) {
 		String filterQuery = filters.isEmpty() ? "" : "AND poitype IN (" + filters.stream().map(s -> "'" + s + "'").collect(Collectors.joining(", ")) + ")";
```

Comment:
Please avoid using **user-defined** `filters` inside string-formed SQL query. It is vulnerable for SQL injections. Using any strings as "'" + string "'" is an awful approach of forming SQL queries. Use SQL placeholders instead.